my-wireframe/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wire frame</title>
+    <link rel="stylesheet" href="css/index.css">
+</head>
+
+<body>
+    <header>
+        <h1>Git Essentials: A Guide to Version Control for Developers</h1>
+        <p><b>Git Essentials</b> is a beginner-friendly site  designed to explain the fundamental concepts of
+            Git, the widely-used version control system. Learn what Git is, why developers rely on it for tracking
+            changes, and how branching works to enable collaborative development. Whether you're new to Git or looking
+            to refresh your knowledge, this guide offers clear, concise answers to essential Git questions.</p>
+    </header>
+
+    <div class="articles">
+        <article>
+            <img src="images/git.webp"></img>
```

Comment:
Is this a valid img tag? How would you find out?

Your website is deployed as a PR preview here https://deploy-preview-109--cyf-ufd.netlify.app/my-wireframe/

Try running Lighthouse on the page. And sending it to the [validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-109--cyf-ufd.netlify.app%2Fmy-wireframe%2F).  What advice do these tools give you?

Reply:
I have added the alt tag for each image tag and reomves the closing. Now the Nu Html Checker is happy with it.